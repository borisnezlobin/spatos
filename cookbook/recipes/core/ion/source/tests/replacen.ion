let input = "one two one two"
echo $replacen(input "one" "three" 0)
echo $replacen(input "one" "" 2)
