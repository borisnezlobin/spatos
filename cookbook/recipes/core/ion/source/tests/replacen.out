one two one two
 two  two
